integreat_cms/cms/migrations/0066_update_poi_translation_status.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations
+
+from ..constants import status
+
+
+# pylint: disable=unused-argument
+def update_poi_translation_status(apps, schema_editor):
+    """
+    Update poi translation status to draft for pois without the default public translation
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    LanguageTreeNode = apps.get_model("cms", "LanguageTreeNode")
+    POITranslation = apps.get_model("cms", "POITranslation")
+
+    for translation in POITranslation.objects.filter(status=status.PUBLIC):
+        default_language = LanguageTreeNode.objects.get(
+            region_id=translation.poi.region.id, active=True, parent_id=None
+        ).language
+
+        default_translation = (
+            POITranslation.objects.filter(
+                poi_id=translation.poi.id, language_id=default_language.id
+            )
+            .order_by("-version")
+            .first()
+        )
+
+        if default_translation.status != status.PUBLIC:
+            translation.status = status.DRAFT
+            translation.save()
```

Comment:
```suggestion
    Region = apps.get_model("cms", "Region")
    POITranslation = apps.get_model("cms", "POITranslation")
    for region in Region.objects.all():
        default_language = region.language_tree_nodes.values_list(
            "language", flat=True
        ).first()
        POITranslation.objects.filter(
            poi__region=region,
            status=status.PUBLIC,
        ).exclude(
            poi__in=region.pois.filter(
                translations__language=default_language,
                translations__status=status.PUBLIC,
            )
        ).update(
            status=status.DRAFT
        )
```
I think we can optimize this a bit to achieve the same thing in fewer sql queries (your solution did multiple queries for each translation object, which can get pretty significant in the production database).

Reply:
Thank you!
Do we want to exclude regions without locations from the iteration?
I assume we have many of them.
I will add this to the filter.